### PR TITLE
Add smoke tests

### PR DESCRIPTION
**Pull request by ErikSchierboom**
This PR adds smoke tests to verify that the representer works well when run as a Docker container.
Unfortunately, there is an error which looks to be related to the code attempting to do a network request, whch it can't (we run our docker containers without networking).

@sanderploegsma would you mind taking a look?

### CODE REVIEW

File: tests/normalize-whitespace/.meta/src/reference/java/Anagram.java
Comment (sanderploegsma) [inline tests/normalize-whitespace/.meta/src/reference/java/Anagram.java:1]: Shall we remove the .meta files not used by the tests?

**Review by sanderploegsma — APPROVED** [review]
General setup looks good to me. I noticed that the expected representations are empty, I assume that's because it currently fails to run?

File: tests/normalize-whitespace/.meta/src/reference/java/Anagram.java
Comment (ErikSchierboom) [reply tests/normalize-whitespace/.meta/src/reference/java/Anagram.java:1]: We could, yes. I use them in the C# representer to found out any non-standard solution files that were submitted, but I guess that's easier for Java due to the fixed directory structure? But then again, you'd want to exclude invalidator or editor files, so maybe this can be used later on.

**Comment by ErikSchierboom** [comment]
>  noticed that the expected representations are empty, I assume that's because it currently fails to run?

Correct.

File: bin/run-tests.sh
Comment (sanderploegsma) [inline bin/run-tests.sh:21]: I'm not sure that this is a problem, but according to the [representer docs](https://exercism.org/docs/building/tooling/representers/interface) the representer can expect to be called with `slug`, `path/to/src`, `path/to/output`. This script breaks that interface, as it passes the name of the test directory as if it's a slug.

Comment (ErikSchierboom) [reply bin/run-tests.sh:21]: True, but it usually doesn't matter in practice (and it doesn't here I think).

Comment (sanderploegsma) [reply bin/run-tests.sh:21]: Fair enough!

**Comment by sanderploegsma** [comment]
I'm OK with merging it, but it needs a review from @exercism/maintainers-admin before I can do that.

**Comment by ErikSchierboom** [comment]
Well, I have special powers :)

**Pull request merged**
